### PR TITLE
move from python 2 to python 3

### PR DESCRIPTION
**Pull request by flying-pi**

### CODE REVIEW

File: bridge_adaptivity/.codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Serves for configuration of CodeClimate (automated code review for test coverage, complexity, duplication, etc.)
```
Comment (idegtiarov) [inline bridge_adaptivity/.codeclimate.yml]: This config file is redundant remove it please

File: bridge_adaptivity/api/backends/openedx.py [hunk 2]
```diff
@@ -210,7 +210,7 @@ def get_available_courses(source_id=None):
 
 def add_to_dict(data, **kwargs):
     """Add key and value to dict only if this pair not exist yet in data dict."""
-    for k, v in kwargs.items():
+    for k, v in list(kwargs.items()):
```
Comment (idegtiarov) [inline bridge_adaptivity/api/backends/openedx.py]: why do you use `list` instead of `generator`?

File: bridge_adaptivity/api/backends/openedx.py [hunk 3]
```diff
@@ -231,7 +231,7 @@ def apply_data_filter(data, filters=None, **kwargs):
 
     filtered_data = []
     for resource in data:
-        filtered_resource = {k: v for k, v in resource.items() if k in filters}
+        filtered_resource = {k: v for k, v in list(resource.items()) if k in filters}
```
Comment (idegtiarov) [inline bridge_adaptivity/api/backends/openedx.py]: ditto?

File: bridge_adaptivity/docker-compose.yml [hunk 4]
```diff
@@ -25,7 +25,7 @@ services:
 
   # Celery worker
   worker:
-    image: bridgeadaptivity_bridge
+    image: bridge_adaptivity_bridge
```
Comment (idegtiarov) [inline bridge_adaptivity/docker-compose.yml]: why does this change be added?

File: bridge_adaptivity/docker-compose_local.yml [hunk 5]
```diff
@@ -26,7 +26,7 @@ services:
 
    # Celery worker
   worker:
-    image: bridgeadaptivity_bridge
+    image: bridge_adaptivity_bridge
```
Comment (idegtiarov) [inline bridge_adaptivity/docker-compose_local.yml]: ditto

File: bridge_adaptivity/module/base_views.py [hunk 6]
```diff
@@ -23,7 +23,7 @@ class BaseGroupView(OnlyMyObjectsMixin, BackURLMixin):
     def get_form_kwargs(self):
         kwargs = super(BaseGroupView, self).get_form_kwargs()
         model_field_names = [f.name for f in CollectionGroup._meta.fields]
-        filtered_get = {key: value for key, value in self.request.GET.items() if key in model_field_names}
+        filtered_get = {key: value for key, value in list(self.request.GET.items()) if key in model_field_names}
```
Comment (idegtiarov) [inline bridge_adaptivity/module/base_views.py]: Why do you use `list` insteasd of `generator`

File: bridge_adaptivity/module/models.py [hunk 7]
```diff
@@ -62,7 +62,7 @@ def _load_cls_from_applicable_module(module_path, mod_name, class_startswith=Non
 This hack is done to pass policy.summary_text and policy.detail_text to select policy widget's template, where these
 variables are used to show popover message with description about each policy (bootstrap3 JS popover function).
 """
-GRADING_POLICY_CHOICES = ((k, v) for k, v in GRADING_POLICY_NAME_TO_CLS.items())
+GRADING_POLICY_CHOICES = ((k, v) for k, v in list(GRADING_POLICY_NAME_TO_CLS.items()))
```
Comment (idegtiarov) [inline bridge_adaptivity/module/models.py]: ditto

File: bridge_adaptivity/module/tests/test_grading_policy.py [hunk 8]
```diff
@@ -97,7 +97,7 @@ def test_policy_math(
         mock_points_earned_get_points_earned_trials_count.return_value = trials_count, points_earned
         mock_points_earned_trials_count.return_value = trials_count, points_earned
 
-        POLICY_CLS_TO_NAME = {v: k for k, v in GRADING_POLICY_NAME_TO_CLS.items()}
+        POLICY_CLS_TO_NAME = {v: k for k, v in list(GRADING_POLICY_NAME_TO_CLS.items())}
```
Comment (idegtiarov) [inline bridge_adaptivity/module/tests/test_grading_policy.py]: ditto

File: bridge_adaptivity/module/tests/test_views.py [hunk 9]
```diff
@@ -24,7 +24,7 @@ class BridgeTestCase(TestCase):
 
     def add_prefix(self, prefix='', data={}):
         """Add prefix to form data dict, which will be send as POST or GET to view."""
-        return {"{}-{}".format(prefix, k): v for k, v in data.items()}
+        return {"{}-{}".format(prefix, k): v for k, v in list(data.items())}
```
Comment (idegtiarov) [inline bridge_adaptivity/module/tests/test_views.py]: ditto

File: .gitignore [hunk 10]
```diff
@@ -19,5 +19,4 @@ secure.config
 ***.log
 ***.log.*
 ***.egg*
-.tox
```
Comment (idegtiarov) [inline .gitignore]: It is incorrect, revert this change please

File: bridge_adaptivity/tox.ini [hunk 11]
```diff
@@ -0,0 +1,33 @@
+[tox]
```
Comment (idegtiarov) [inline bridge_adaptivity/tox.ini]: remove this file it already exists in the one layer upper.

File: bridge_adaptivity/.codeclimate.yml (hunk 1 above)
Comment (flying-pi) [reply bridge_adaptivity/.codeclimate.yml]: done

File: bridge_adaptivity/docker-compose.yml (hunk 4 above)
Comment (flying-pi) [reply bridge_adaptivity/docker-compose.yml]: because it was a mistake which causes an error in building an image for celery worker.

File: bridge_adaptivity/docker-compose_local.yml (hunk 5 above)
Comment (flying-pi) [reply bridge_adaptivity/docker-compose_local.yml]: ditto

File: .gitignore (hunk 10 above)
Comment (flying-pi) [reply .gitignore]: done
